modules/apps/transfer/types/ack.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+import (
+	"fmt"
+
+	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+)
+
+// NewForwardErrorAcknowledgement returns a new error acknowledgement with path forwarding information.
+func NewForwardErrorAcknowledgement(packet channeltypes.Packet, ack channeltypes.Acknowledgement) channeltypes.Acknowledgement {
+	ackErr := fmt.Sprintf("forwarding packet failed on %s/%s: %s", packet.GetSourcePort(), packet.GetSourceChannel(), ack.GetError())
```

Comment:
Do you think it makes sense to be explicit and add "source" in the message?

```suggestion
	ackErr := fmt.Sprintf("forwarding packet failed on source %s/%s: %s", packet.GetSourcePort(), packet.GetSourceChannel(), ack.GetError())
```

Reply:
I think a timeout is implicitly source. Fine to leave as is I think! Applied the other suggestion to align string with error ack